packages/axios-extension/src/abap/adt-catalog/handlers/transport.ts

```diff
@@ -2,6 +2,7 @@ import XmlParser from 'fast-xml-parser';
 import * as xpath from 'xpath';
 import { DOMParser } from '@xmldom/xmldom';
 import type { Logger } from '@sap-ux/logger';
+import type { TransportRequest } from 'abap/types';
```

Comment:
Do we not need to use `./abap/types`? Do we have some sort of path mapping to make this work?